cmd/thanos/compact.go

```diff
@@ -121,6 +122,9 @@ func registerCompact(m map[string]setupFunc, app *kingpin.Application) {
 	compactionConcurrency := cmd.Flag("compact.concurrency", "Number of goroutines to use when compacting groups.").
 		Default("1").Int()
 
+	deleteDelay := modelDuration(cmd.Flag("delete-delay", "Time before a block marked for deletion is deleted from bucket").
```

Comment:
You elaborated on bucket so much, but not here?

Reply:
I will add.

Reply:
I have added an explanation for this. Kindly check if that will suffice.

Reply:
I am marking this as resolved since there was feedback provided in this thread: https://github.com/thanos-io/thanos/pull/2136#discussion_r387916352